src/components/PersonaCard/PersonaCard.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react';
+import { PersonaCard as PersonaCardType } from 'global/graphqls/schema';
+
+import { CardBody } from 'pages/CreatePersona/CreateCard/CreateCard.styles';
+import { BackgroundPlaceholder } from 'components/BackgroundPlaceholder/BackgroundPlaceholder';
+import { PersonaCircle, PersonaCircleWrapper } from 'components/PersonaCircle/PersonaCircle';
+import { Card } from 'components/Card/Card';
+
+type Props = {
+  uuid?: string;
+  card: PersonaCardType;
+};
+
+export const PersonaCard: FC<Props> = ({ uuid, card }) => (
+  <Card mt={31} mb={40} position="relative" key={uuid}>
```

Comment:
Hmm if uuid is an optional You should provide something alternative for a key, like v4() or something.